salt/metalk8s/map.jinja

```diff
@@ -55,8 +194,21 @@
 }, merge=defaults.get('containerd')) %}
 
 {% set kubelet = salt['grains.filter_by']({
-  'default': {}
-}, merge=defaults.get('kubelet')) %}
+  'Ubuntu': {
+    'service': {
+      'options': {
+        'resolv-conf': '/run/systemd/resolve/resolv.conf'
+      }
+    }
+  },
+  'CentOS': {
+    'service': {
+      'options': {
+        'resolv-conf': '/etc/resolv.conf'
+      }
+    }
+  }
+}, grain='os', merge=defaults.get('kubelet')) %}
```

Comment:
Please use `osfamily` (or just remove `grain='os'` as `osfamily` is the default value) everywhere or @alexandre-allard-scality  change for redhat will be broken in 2.4